src/openms/source/ANALYSIS/ID/AccurateMassSearchEngine.cpp

```diff
@@ -116,6 +116,11 @@ namespace OpenMS
     return name_;
   }
 
+  int AdductInfo::getMolMultiplier() const
+  {
+    return mol_multiplier_;
```

Comment:
initialized to 1 ?

Reply:
As far as I have seen it is always initialized to 1 in the `parseAdductString` function.
https://github.com/OpenMS/OpenMS/blob/e10e86f6bccb089bbe422885891d64b179ab1283/src/openms/source/ANALYSIS/ID/AccurateMassSearchEngine.cpp#L129
https://github.com/OpenMS/OpenMS/blob/e10e86f6bccb089bbe422885891d64b179ab1283/src/openms/source/ANALYSIS/ID/AccurateMassSearchEngine.cpp#L207

Reply:
yes but there are other ways to create an AdductInfo object. Please also make sure that methods that set the mol_multiplier_ check for 0 as this is invalid e.g., https://github.com/OpenMS/OpenMS/blob/e10e86f6bccb089bbe422885891d64b179ab1283/src/openms/source/ANALYSIS/ID/AccurateMassSearchEngine.cpp#L52

Reply:
Should be sufficient to throw an error in the constructor if it is 0, I guess

```
    if (mol_multiplier_ == 0)
    {
      throw Exception::InvalidParameter(__FILE__, __LINE__, OPENMS_PRETTY_FUNCTION, "Molecule multiplier of 0 is not allowed for an adduct (" + ef_.toString() + ")");
    }
```